client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -394,7 +395,10 @@ const PatternAssembler = ( {
 							headerHtml: header?.html,
 							footerHtml: footer?.html,
 							globalStyles: syncedGlobalStylesUserConfig,
+							// Newly created sites should reset the starter content created from the default Headstart annotation
 							shouldResetContent: isNewSite,
+							// Also, new sites except for virtual themes set the option wpcom_site_setup=assembler
+							setSiteSetupOption: isNewSite && ! design.is_virtual ? 'assembler' : '',
```

Comment:
[nit] The naming `setSiteSetupOption` seems like a function instead of a value, would `siteSetupOption` be enough? What do you think?

Reply:
Sure that works too 👍